internal/assets/markdown-breaking-template.md

```diff
@@ -0,0 +1,14 @@
+## {{.Version}} [{{.Repo}}-{{.Version}}-breaking-changes]
```

Comment:
nit : Use `.tmpl` as the file extension (e.g. `markdown-breaking-template.md.tmpl`) for these templates.